### PR TITLE
feat: Plug'n'Play

### PR DESCRIPTION
**Pull request by zkochan**

### CODE REVIEW

**Comment by ExE-Boss** [comment]
`@berry/pnp` is being renamed to `@yarnpkg/pnp` (https://github.com/yarnpkg/berry/pull/395)

**Comment by ExE-Boss** [comment]
See also: #1628

**Comment by ExE-Boss** [comment]
I think some of these pre‑requisites should be factored out into their own PRs:
- #2032

File: packages/supi/src/install/index.ts [hunk 1]
```diff
@@ -893,7 +897,10 @@ async function installInContext (
 
   await opts.storeController.close()
 
-  return importersToLink.map(({ manifest, prefix }) => ({ prefix, manifest }))
+  return {
+    importers: importersToLink.map(({ manifest, prefix }) => ({ prefix, manifest })),
+    wantedLockfile: ctx.wantedLockfile,
```
Comment (ExE-Boss) [inline packages/supi/src/install/index.ts]: This looks unused to me, and seems to be the reason why the build is failing.

**Comment by zkochan** [comment]
I'll resume work on this PR soon.

**Pull request merged**